### DISCUSSION TITLE
How to include specialization on the back cover?

### DISCUSSION THREAD

**Discussion by MathMachado** [joaomlourenco/novathesis#351] [Q&A]
Hello all. I need to add the specialization on the back cover of my thesis. My PhD is in Statistics and Risk Management and the specialization is Stochastic Processes. The expected result would be (bottom of the back cover):

PhD in Statistics and Risk Management
Specialization in Stochastic Processes

NOVA University Lisbon
May, 2022

How to do that?

**Comment by joaomlourenco** [comment]
Humm… Why in the back cover?  That seems a strange place to add it!
Why don't you add it to front page as
```
DOCTORATE IN STATISTICS AND RISK MANAGEMENT
SPECIALIZATION IN STOCHASTIC PROCESSES

NOVA University Lisbon
May, 2022
```
This seems more standard to me!  And also much easier! 😃
